utils/src/ooinstall/cli_installer.py

```diff
@@ -871,7 +872,16 @@ def install(ctx, force):
     if len(oo_cfg.calc_missing_facts()) > 0:
         confirm_hosts_facts(oo_cfg, callback_facts)
 
+    # Write quick installer config file to disk:
     oo_cfg.save_to_disk()
+    # Write ansible inventory file to disk:
+    inventory_file = openshift_ansible.generate_inventory(hosts_to_run_on)
+
+    if gen_inventory:
+        click.echo()
+        click.echo("Generated ansible inventory: %s" % inventory_file)
+        click.echo()
+        sys.exit(0)
```

Comment:
Any objections to displaying the path to the generated inventory for all deployments and just having the new flag only generate the inventory?